diff --git a/tests/unit/latency-monitor.tcl b/tests/unit/latency-monitor.tcl
--- a/tests/unit/latency-monitor.tcl
+++ b/tests/unit/latency-monitor.tcl
@@ -96,3 +96,3 @@ tags {"needs:debug"} {
         set min 250
-        set max 450
+        set max 950
         foreach event $res {
@@ -119,3 +119,3 @@ tags {"needs:debug"} {
             if {!$::no_latency} {
-                assert {$max >= 450 & $max <= 650}
+                assert {$max >= 450 & $max <= 1150}
                 assert {$time == $last_time}
